_episodes/09-cmorization.md

```diff
@@ -2,6 +2,7 @@
 title: "CMORization: adding new datasets to ESMValTool"
 teaching: 15
 exercises: 45
+compatibility: ESMValTool v.2.5.0
```

Comment:
That should be v.2.6.0. Rémi wrote that recently.

Reply:
Ok. I can change this.

Reply:
changed.